loleaflet/src/layer/tile/TileLayer.js

```diff
@@ -3935,12 +3935,24 @@ L.TileLayer = L.GridLayer.extend({
 	},
 
 	getCommentWizardStructure: function(menuStructure) {
+
+		var insertButton = '<button id="insert_comment">+</button>';
+		var title = '<span class="undefined">'+ _('Comment') +'</span>';
```

Comment:
I think it would be better to build fully functional DOM nodes here and just pass them.
```var customTitleBar = L.DomUtil.create('div', 'mobile-wizard custom-title-bar');
var title = L.DomUtil.create('span', '', customTitleBar);
title.innerText = _('Comment');
var button = L.DomUtil.createWithId('button', 'insert_comment', customTitleBar);
button.onclick = map.insertComment.bind(map);
```